File: 2nd Year/2nd Semester/MPP/PROIECT-JAVA/Laborator/TriatlonNetworking/src/main/java/triatlon/network/dto/RezultatDTO.java
package triatlon.network.dto;

import java.io.Serializable;

public class RezultatDTO implements Serializable {
    private Integer id;
    private ParticipantDTO participant;
    private ArbitruDTO arbitru;
    private TipProbaDTO tipProba;
    private int punctaj;

    public RezultatDTO(Integer id, ParticipantDTO participant, ArbitruDTO arbitru, TipProbaDTO tipProba, int punctaj) {
        this.id = id;
        this.participant = participant;
        this.arbitru = arbitru;
        this.tipProba = tipProba;
        this.punctaj = punctaj;
    }

    public Integer getId() {
        return id;
    }

    public ParticipantDTO getParticipant() {
        return participant;
    }

    public ArbitruDTO getArbitru() {
        return arbitru;
    }

    public TipProbaDTO getTipProba() {
        return tipProba;
    }

    public int getPunctaj() {
        return punctaj;
    }
}

File: 2nd Year/2nd Semester/MPP/PROIECT-C#/Lab1/TriatlonNetworking/dto/RezultatDTO.cs
using System;

namespace TriatlonNetworking.dto
{
    [Serializable]
    public class RezultatDTO
    {
        public int? Id { get; set; }
        public ParticipantDTO Participant { get; set; }
        public ArbitruDTO Arbitru { get; set; }
        public TipProbaDTO TipProba { get; set; }
        public int Punctaj { get; set; }

        public RezultatDTO(int? id, ParticipantDTO participant, ArbitruDTO arbitru, TipProbaDTO tipProba, int punctaj)
        {
            Id = id;
            Participant = participant;
            Arbitru = arbitru;
            TipProba = tipProba;
            Punctaj = punctaj;
        }
    }
}